user/tarsnap/template.py

```diff
@@ -0,0 +1,27 @@
+pkgname = "tarsnap"
+pkgver = "1.0.40"
+pkgrel = 0
+build_style = "configure"
+configure_args = [
+    "--prefix=/usr",
+]
+makedepends = [
+    "openssl-devel",
+    "zlib-ng-compat-devel",
+    "e2fsprogs-devel",
+]
+pkgdesc = "Online backups for the truly paranoid"
+maintainer = "Nasado <hi@nasado.name>"
+license = "custom:tarsnap"
+url = "https://www.tarsnap.com"
+source = f"{url}/download/tarsnap-autoconf-{pkgver}.tgz"
+sha256 = "bccae5380c1c1d6be25dccfb7c2eaa8364ba3401aafaee61e3c5574203c27fd5"
+
+
+def post_install(self):
+    self.mkdir(self.destdir / "etc")
+    self.mv(
+        self.destdir / "usr/etc/tarsnap.conf.sample",
+        self.destdir / "etc/tarsnap.conf.sample",
+    )
```

Comment:
self.rename

Reply:
though this should be in usr/share/doc or something

Reply:
although that would also make it not installed in main package so not sure either..